### PR TITLE
Di but Better

### PR DESCRIPTION
**Pull request by SrLicht**
Improvements ?
- I don't like to compile code, so I made a github action for that, honestly I copied a little (a lot) from the Exiled repository.
- csprojec was changed to SDK, I don't know if this is really an improvement. I also fixed the references, sorry for breaking them.
- Support of CategoryID instead of ChannelID to register the channels that can use commands. Servers with many channels should thank me.
- Now all command responses are embed messages.
  - I made a setting in DI plugin to change the embed color.
image
  
 Things I plan to change:
 - Make the embed of the !di playerlist command have the time of the game in the embed title.

https://www.youtube.com/watch?v=5xooMXyleXM

### CODE REVIEW

**Comment by joker-119** [comment]
> when?

I told pietro to review this ages ago but he seems to have fallen off the face of the earth

**Comment by SrLicht** [comment]
> > when?
> 
> I told pietro to review this ages ago but he seems to have fallen off the face of the earth
Anyway this is broken with the current version of DI, I would ask you to give me a week or two to fix it and optimize it. Although I don't think it is necessary pietro is dead, I have all the time I want. :trollface:

File: build.ps1 [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    $Version = $Version.'#text'.Trim()
+    return $Version
+}
+
+# Restore projects
+Execute 'dotnet restore'
+# Build projects
+Execute 'dotnet build'
+# Build a NuGet package if needed
+if ($BuildNuGet) {
```
Comment (SrLicht) [inline build.ps1:40]: You should remove this.

**Comment by xRoier** [comment]
This PR should be closed already lol

**Pull request closed without merging**